ci/docs/build.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+set -e
+
+export HOME=$WORKSPACE
+export DOCS_WORKSPACE=$WORKSPACE/docs
+export NIGHTLY_VERSION=$(echo $BRANCH_VERSION | awk -F. '{print $2}')
+
+export CUDA_REL=${CUDA_VERSION%.*}
+export CUDA_SHORT=${CUDA_REL//./}
+export CLX_HOME=$WORKSPACE/clx_build
+
+# Switch to project root; also root of repo checkout
+cd $WORKSPACE
+export GIT_DESCRIBE_TAG=`git describe --tags`
+export MINOR_VERSION=`echo $GIT_DESCRIBE_TAG | grep -o -E '([0-9]+\.[0-9]+)'`
+
+logger "Check environment..."
+env
+
+logger "Check GPU usage..."
```

Comment:
I noticed there were some `pip` dependencies from the original CI script that were removed here. I know at least some of those are required for the build process (like `jq` and `beautifulsoup4` for the HTML customization). We'll need to put those back before this can be merged.

Reply:
Removed `ci/docs/build.sh` until gpuci moves to running docs builds from repos.